wp-content/themes/core/components/routes/search/css/search.pcss

```diff
@@ -0,0 +1,114 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Route: Search Results
+ *
+ * ----------------------------------------------------------------------------- */
+
+.search-results__container {
+	max-width: var(--grid-width-staggered-double);
+	margin: 0 auto;
+	padding-top: var(--spacer-70);
+	padding-bottom: var(--spacer-70);
+}
+
+.search-results__title {
+	margin-bottom: var(--spacer-20);
+
+	@media (--viewport-medium) {
+		margin-bottom: var(--spacer-30);
+	}
+}
+
+.search-results__form {
+	margin-bottom: var(--spacer-40);
+
+	@media (--viewport-medium) {
+		margin-bottom: var(--spacer-70);
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Search Results: Summary
+ * ----------------------------------------------------------------------------- */
+
+.search-results__summary {
+	margin-bottom: var(--spacer-20);
+	line-height: 1.75;
+	font-weight: 700;
+	letter-spacing: 0.1px;
```

Comment:
Any chance we can use an existing typography mixin for this?

If this really is a unique style and it needs to be unique, let's add a mixin and helper class for it in the typography pcss partials. We're trying to avoid any one-off typeography throughout the components. Additionally, please run it by design to make sure it gets added to the Figma Style Guide for typography styles.